editor/scene_tree_dock.cpp

```diff
@@ -3358,7 +3358,7 @@ SceneTreeDock::SceneTreeDock(Node *p_scene_root, EditorSelection *p_editor_selec
 	vbc->add_child(filter_hbc);
 	filter = memnew(LineEdit);
 	filter->set_h_size_flags(SIZE_EXPAND_FILL);
-	filter->set_placeholder(TTR("Filter nodes"));
+	filter->set_placeholder(TTR("Filter nodes (by name or type:Node)"));
```

Comment:
The placeholder text is too long to be full shown in a regular setup. With the default layout, it looks like this on my screen:

![ksnip_20220413-095657](https://user-images.githubusercontent.com/372476/163084482-f6a28564-d1b5-41b7-a4bb-8c1181516e07.png)

Reply:
I wonder if there is any example in the godot sourcecode that control's properties changes when size changes?

Reply:
In this case we don't want to make the SceneTreeDock much wider than it already is though.

So it's better to keep the placeholder short. Additional documentation on advanced options could maybe go in a tooltip.

Reply:
> In this case we don't want to make the SceneTreeDock much wider than it already is though.
> 
> So it's better to keep the placeholder short. Additional documentation on advanced options could maybe go in a tooltip.

So, should I just keep it original text "Filter Nodes" ? 
The textbox seems have no tooltip.